spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1451,6 +1451,13 @@ A client for the Spring Cloud Stream schema registry can be configured using the
   }
 ----
 
+[NOTE]
+====
+A decorator bean of type `CachingRegistryClient` is available if you want to cache responses from `fetch` operations. In your application
```

Comment:
We follow the "one sentence per line pattern{ in our documentation. See http://www.vogella.com/tutorials/AsciiDoc/article.html#one-sentence-per-line .